Library/Formula/onepass.rb

```diff
@@ -14,47 +14,33 @@ class Onepass < Formula
   end
 
   depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "swig" => :build
   depends_on "openssl" # For M2Crypto
 
-  # For vendored Swig
-  depends_on "pcre" => :build
-
-  # Homebrew's swig breaks M2Crypto due to upstream's undermaintained status.
-  # https://github.com/swig/swig/issues/344
-  # https://github.com/martinpaljak/M2Crypto/issues/60
-  resource "swig304" do
-    url "https://downloads.sourceforge.net/project/swig/swig/swig-3.0.4/swig-3.0.4.tar.gz"
-    sha256 "410ffa80ef5535244b500933d70c1b65206333b546ca5a6c89373afb65413795"
+  resource "m2crypto" do
+    url "https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.22.6rc4.tar.gz"
```

Comment:
As discussed in saltstack PR, this is a stable release :) https://pypi.python.org/pypi/M2Crypto